HabitTracker.ConsoleApp-DreamFXX/Program.cs

```diff
@@ -0,0 +1,366 @@
+        string? habitUnit = Console.ReadLine();
+
+        using (var connection = new SQLiteConnection(connectionString))
+        {
+            connection.Open();
+            var tableCmd = connection.CreateCommand();
+            tableCmd.CommandText = $"INSERT into Habits (Name, Unit) VALUES ('{habitName}', '{habitUnit}')";
+
+            tableCmd.ExecuteNonQuery();
+            connection.Close();
+        }
+        Console.WriteLine($"New Habit named - '{habitName}' was sucessfully added");
+    }
+
+    static void ViewHabits()
+    {
+        using(var connection = new SQLiteConnection(connectionString))
+        {
+            connection.Open();
+            var tableCmd = connection.CreateCommand();
+            tableCmd.CommandText = "SELECT * FROM Habits";
+
+            using(var reader = tableCmd.ExecuteReader())
+            {
+                Console.WriteLine("\nHabits available to track in this App: ");
+
+                while (reader.Read())
+                {
+                    int id = reader.GetInt32(0);
+                    string name = reader.GetString(1);
+                    string unit = reader.GetString(2);
+                    Console.WriteLine($"{id}. {name} ({unit})");
+                }
+            }
+            connection.Close();
+        }
+    }
+
+    static void FillDatatables()
+    {
+        using(var connection = new SQLiteConnection(connectionString))
+        {
+            connection.Open();
+            var tableCmd = connection.CreateCommand();
+
+            tableCmd.CommandText = "SELECT Id FROM Habits LIMIT 1";
+            var habitIdObj = tableCmd.ExecuteScalar();
+
+            int habitId;
+            if (habitIdObj == null)
+            {
+                tableCmd.CommandText = "INSERT INTO Habits (Name, Unit) VALUES ('Cycling', 'kilometers')";
+                tableCmd.ExecuteNonQuery();
+
+                tableCmd.CommandText = "SELECT Id FROM Habits LIMIT 1";
+                habitId = Convert.ToInt32(tableCmd.ExecuteScalar());
+            }
+            else
+            {
+                habitId = Convert.ToInt32(habitIdObj);
+            }
+
+            Random random = new Random();
+
+            for (int i = 0; i < 100; i++)
+            {
+                var date = DateTime.Today.AddDays(-random.Next(0, 365)).ToString("dd-MM-yy");
+
+                DateTime datetime = DateTime.Now;
+                var time = TimeOnly.FromDateTime(datetime).ToString();
+
+                int quantity = random.Next(1, 100);
+                tableCmd.CommandText = $"INSERT INTO HabitRecords (HabitId, Date, Time, Quantity) VALUES({habitId}, '{date}', '{time}', {quantity})";
+                tableCmd.ExecuteNonQuery();
+            }
+            connection.Close();
+        }
+        Console.WriteLine("Testing records (100) were sucessfully created and added to their specified tables.");
+    }
+
+    // Properties class
+
+    public class HabitRecord
```

Comment:
🟠 Object Orientated Programming

💡 Again, I know the project scope allows for this. But I would really of liked to see this in it's own file `HabitRecord.cs`

🔧 Refactor into it's own file.